README.md

```diff
@@ -34,35 +34,57 @@ You can either [clone the repository from GitHub](https://github.com/jtebert/kil
 
 ### Dependencies
 
-[HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5), [SFML](https://www.sfml-dev.org/index.php), and [OpenMP](https://www.openmp.org/). Install on Ubuntu-like systems with: `sudo apt install libsfml-dev libhdf5-dev libomp-dev`
+[HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5), [SFML](https://www.sfml-dev.org/index.php), and [OpenMP](https://www.openmp.org/). Install on Ubuntu-like systems with:
 
-[nlohmann/json](https://github.com/nlohmann/json): Add [`json.hpp`](https://github.com/nlohmann/json/releases/download/v3.5.0/json.hpp) to the `include/` folder. **Note:** This links to the most recent release (3.5) that has been verified for compatibility. You can find other [versions and release notes on Github](https://github.com/nlohmann/json/releases).
+    sudo apt install libsfml-dev libhdf5-dev libomp-dev
 
 ### Compile
 
-Create a static library at `bin/libKilosim.a`:
+To compile, copy and paste the following:
 
-    make
-or
+    mkdir build                                #Create a build directory
+    cd build                                   #Move into build directory
+    cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .. #Configure for your system
+    make                                       #Build all
+    make install                               #Install files
+    cd ..                                      #Move back to source directory
 
-    make static
+Alternatively, you can run:
 
-Create an executable at `bin/kilosim` using the example `MyKilobot.cpp` and `test.cpp`:
+    make kilosim                        #Build library
+    make examples                       #Build examples
+    make kilosim_docs                   #Build documentation
 
-    make exec
+Flags:
 
-**NOTE:** This is tested with g++ 7.3 but currently fails (for as-of-yet unclear reasons) with g++ 8.1
+ * `-DSANITIZE_ADDRESS=On`: Use address sanitizer to check memory safety
+
+Use flags like so:
+
+    cmake -DSANITIZE_ADDRESS=ON ..
+
+Compatibility:
+
+ * g++ 7.3 - works
+ * g++ 8.1 - fails due to compiler issue(?)
+ * g++ 8.3 - works 
 
 ### Run
 
-Run the executable generated by `make exec`
+Run the executable generated by `make tests`
```

Comment:
```
make: *** No rule to make target 'tests'.  Stop.
```

Should this be "Run the executable generated by `make examples`"?

Reply:
Yes.